exonum-java-binding/fakes/src/test/java/com/exonum/binding/fakes/services/service/TestServiceIntegrationTest.java

```diff
@@ -37,7 +37,7 @@
 
 class TestServiceIntegrationTest {
 
-  @Disabled // FIXME: disabled until ECR-XYZ.
+  @Disabled // FIXME: disabled until ECR-3320.
```

Comment:
ECR-3320 — new proof format?

Reply:
It fails because the current proofs code is broken thus blocked by this ticket.